cli/src/semgrep/commands/scan.py

```diff
@@ -348,6 +348,15 @@ def convert(
             skipped. If set to 0 will not have limit. Defaults to 3.
         """,
     ),
+    optgroup.option(
+        "--pro-timeout",
+        type=int,
+        help=f"""
+            Maximum time to spend on preprocessing for pro features. If set to 0
+            will not have time limit. Defaults to 0 s unless the DeepSemgrep toggle
```

Comment:
Maybe App should rename the toggle to `Semgrep Pro` (even if it's just in the UI, that should be easy) so that we don't need to refer to "DeepSemgrep" here either. I also find  a bit confusing to refer to the App toggle here. This is the `scan` command command, not the `ci` one.

Reply:
Yeah, I agree that's a bit confusing but I don't know another place to put it